paimon-core/src/main/java/org/apache/paimon/manifest/SimpleFileEntry.java

```diff
@@ -38,6 +38,7 @@ public class SimpleFileEntry implements FileEntry {
     @Nullable private final byte[] embeddedIndex;
     private final BinaryRow minKey;
     private final BinaryRow maxKey;
+    private final String externalPath;
```

Comment:
Don't forget to add Nullable annotation.

Reply:
got it